config/locales/en.yml

```diff
@@ -1,33 +1,12 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t 'hello'
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t('hello') %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at http://guides.rubyonrails.org/i18n.html.
-
 en:
   hello: "Hello world"
+  new_rent_notification_user_text: 'User: '
```

Comment:
We can refactor this a little to be more readable, for example, instead of prefixing each translation with `new_rent_notification`, you can nest all the translations one level.
```yml
en:
  new_rent_notification:
    user_text: 'User: '
    rents_text: 'Rents the book: '
    ....
```

And then you can access the translation with `t('new_rent_notification.user_text')`.What do you think?